src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.service.OperationSelector;
+import core.basesyntax.service.impl.OperationSelectorBalance;
+import core.basesyntax.service.impl.OperationSelectorPurchase;
+import core.basesyntax.service.impl.OperationSelectorReturn;
+import core.basesyntax.service.impl.OperationSelectorSupply;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    @Override
+    public OperationSelector get(String type) {
+        switch (type) {
+            case "b":
+                return new OperationSelectorBalance();
+            case "r":
+                return new OperationSelectorReturn();
+            case "p":
+                return new OperationSelectorPurchase();
+            case "s":
+                return new OperationSelectorSupply();
+            default:
+                throw new RuntimeException("Can't work with this operator");
+        }
+    }
```

Comment:
We should avoid using switch case 
![image](https://user-images.githubusercontent.com/63553735/198062214-fa1a45b0-46b1-474a-9ff5-6c846a6e3450.png)

Reply:
Oh, I missed it in common mistakes, thanks